src/nvim/api/private/converter.c

```diff
@@ -356,7 +356,7 @@ bool object_to_vim(Object obj, typval_T *tv, Error *err)
     char *name =
       (char *)register_cfunc(&nlua_CFunction_func_call, &nlua_CFunction_func_free, state);
     tv->v_type = VAR_FUNC;
-    tv->vval.v_string = xstrdup(name);
+    tv->vval.v_string = name;
```

Comment:
ditto

Reply:
Given the test failures, this seems to be the case.
So I'll mark this as a false positive ?